Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -0,0 +1,2011 @@
+    return trans_assets, human_readable_trans_assets
+
+
+def transform_single_asset_to_hr(asset):
+    """
+    Prepares asset for human readable
+    """
+    hr_asset = []
+    for k, v in asset.items():
+        if isinstance(v, dict):
+            hr_item = v
+            hr_item["Property Name"] = k
+            hr_asset.append(hr_item)
+    return hr_asset
+
+
+def create_single_asset_result_and_enrich_endpoint_dict(
+    asset, endpoint_dict, full_values
+):
+    asset_dict = {"ID": asset.get("id")}
+    for interface in asset.get("interfaces", []):
+        if full_values:
+            endpoint_dict.get("MACAddress").append(interface.get("mac_address"))
+        for ip_address in interface.get("ip_addresses"):
+            endpoint_dict.get("IPAddress").append(ip_address.get("value"))
+    if full_values:
+        if "domain_id" in asset:
+            domain_name = get_domain_name(asset.get("domain_id"))
+            if domain_name:
+                endpoint_dict.get("Domain").append(domain_name)
+    # Adding values found in properties of the asset
+    enrich_dict_using_asset_properties(asset, asset_dict, endpoint_dict, full_values)
+    return asset_dict
+
+
+def enrich_dict_using_asset_properties(asset, asset_dict, endpoint_dict, full_values):
+    for prop in asset.get("properties", []):
+        if prop.get("name") in ASSET_PROPERTIES_NAMES_MAP:
+            asset_dict[ASSET_PROPERTIES_NAMES_MAP[prop.get("name")]] = {
+                "Value": prop.get("value"),
+                "LastUser": prop.get("last_reported_by"),
+            }
+        elif prop.get("name") in ASSET_PROPERTIES_ENDPOINT_NAMES_MAP:
+            endpoint_dict[
+                ASSET_PROPERTIES_ENDPOINT_NAMES_MAP[prop.get("name")]
+            ] = prop.get("value")
+        elif full_values:
+            if prop.get("name") in FULL_ASSET_PROPERTIES_NAMES_MAP:
+                asset_dict[FULL_ASSET_PROPERTIES_NAMES_MAP[prop.get("name")]] = {
+                    "Value": prop.get("value"),
+                    "LastUser": prop.get("last_reported_by"),
+                }
+    return None
+
+
+def create_empty_endpoint_dict(full_values):
+    """
+    Creates an empty endpoint dictionary (for use in other functions)
+    """
+    endpoint_dict = {"IPAddress": [], "OS": []}  # type: dict
+    if full_values:
+        endpoint_dict["MACAddress"] = []
+        endpoint_dict["Domain"] = []
+    return endpoint_dict
+
+
+def get_domain_name(client: QRadarClient, domain_id=None):
+    """
+    Retrieves domain name using domain id
+    """
+    try:
+        query_param = {
+            "query_expression": f"SELECT DOMAINNAME({domain_id}) AS 'Domain name' FROM events GROUP BY 'Domain name'"
+        }
+        search_id = client.search(query_param)["search_id"]
+        time.sleep(5)
+        return (
+            client.get_search_results(search_id)
+            .get("events", [{}])[0]
+            .get("Domain name")
+        )
+    except Exception:
```

Comment:
why are we passing here a general exception without a log?

Reply:
No reason, this was missed since it's is legacy code from v1. Adding it now.